src/util-logopenfile.c

```diff
@@ -416,6 +417,12 @@ int SCConfLogOpenRedis(ConfNode *redis_node, LogFileCtx *log_ctx)
             SCLogError(SC_ERR_MEM_ALLOC, "Unable to allocate redis key command");
             exit(EXIT_FAILURE);
         }
+    } else if (!strcmp(redis_mode, "rlist")) {
+        log_ctx->redis_setup.command = redis_rpush_cmd;
+        if (!log_ctx->redis_setup.command) {
```

Comment:
I removed  these checks while redis refactoring was done at master version, because these are not heap allocated and the error messages looked misleading.